src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -3547,6 +3547,7 @@ class CreateTriggerStatementSegment(BaseSegment):
             "PROCEDURE",
             Ref("FunctionNameIdentifierSegment"),
             Bracketed(Ref("FunctionContentsGrammar", optional=True)),
+            optional=True,
```

Comment:
This looks like the change you made in the other PR, to support SQLite triggers, I think. Is this related to CTEs in CREATE VIEW somehow?

Reply:
Yup reverted that. Check the latest version now.